nixos/modules/services/backup/restic.nix

```diff
@@ -31,6 +31,59 @@ in
           '';
         };
 
+        rcloneOptions = mkOption {
+          type = with types; nullOr (attrsOf (oneOf [ str bool ]));
+          default = null;
+          description = ''
+            Options to pass to rclone to control its behavior.
+            See <link xlink:href="https://rclone.org/docs/#options"/> for
+            available options. When specifying option names, strip the
+            leading <literal>--</literal>. To set a flag such as
+            <literal>--drive-use-trash</literal>, which does not take a value,
+            set the value to <literal>"true"</literal>.
+          '';
+          example = {
+            bwlimit = "10M";
+            drive-use-trash = "true";
+          };
+        };
+
+        rcloneConfig = mkOption {
+          type = with types; nullOr (attrsOf (oneOf [ str bool ]));
```

Comment:
This suggests it can also be a boolean, but the description explicitly says you should assign the `"true"` string. I'd prefer if it'd be possible to assign booleans, and the code would handle converting it to string if needed (and the test making use of it).

Reply:
Yeah, I wasn't sure how to handle flags at first, so I kept going back and forth between using booleans and strings. I fixed this to use `lib.boolToString` when reading boolean values in both `rcloneConfig` and `rcloneOptions`.

Reply:
In the test, I added the `one_file_system` flag to the local remote.